jbmc/src/janalyzer/janalyzer_parse_options.cpp

```diff
@@ -757,8 +759,9 @@ void janalyzer_parse_optionst::help()
     "\n"
     "Domain options:\n"
     " --constants                  constant domain\n"
-    " --intervals                  interval domain\n"
-    " --non-null                   non-null domain\n"
+    " --intervals, --show-intervals\n"
```

Comment:
:bulb: Looks as if there should be a common definition of these options somewhere. Non-blocking.

Reply:
Ack, but actually there's quite a lot more that seems to be duplicated across goto-analyzer and janalyzer. A variant of goto-checker would be nice :-) So I'll leave this as-is for now, but will put it in my backlog.

Reply:
A. There is a lot that could be refactored between the two.  I guess I hadn't because of concern about breaking out-of-tree janalyzer patches.  But, yeah, could look at that post-marking.
B. The `--show-*` options are legacy.  The goal is to "orthogonalize" the options so it is `--show --intervals`.  I would rather these were removed (along with their complex option parsing) than documented.